docs/attribution.mdx

```diff
@@ -19,134 +19,250 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
+- [Sacramento County](https://services1.arcgis.com/5NARefyPVtAeuJPU/ArcGIS/rest/services/Address/FeatureServer/0). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- [San Benito County](https://cosb.maps.arcgis.com/home/item.html?id=f7dd7210ee114965aeeac18e9886daec). Available under [Public domain by judicial interpretation of the Public Records Act](https://casetext.com/case/county-of-santa-clara-v-superior-court-10). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- [San Bernardino County](https://open.sbcounty.gov/datasets/sbcounty::countywide-parcels/about). Available under [Public domain by judicial interpretation of the Public Records Act](https://casetext.com/case/county-of-santa-clara-v-superior-court-10). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- [San Diego County](https://geo.sandag.org/portal/home/item.html?id=b1c2fd749f6847ccae6040cf0f803647). Available under [Public domain by judicial interpretation of the Public Records Act](https://casetext.com/case/county-of-santa-clara-v-superior-court-10). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- [San Joaquin County](https://sjmap.org/GISDataDownload.htm). Available under [Public domain by judicial interpretation of the Public Records Act](https://casetext.com/case/county-of-santa-clara-v-superior-court-10). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- [San Luis Obispo County](https://opendata.slocounty.ca.gov/datasets/SLOCounty::address-points/about). Available under [Public domain by judicial interpretation of the Public Records Act](https://casetext.com/case/county-of-santa-clara-v-superior-court-10). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- [San Mateo County](https://gis.smcgov.org/maps/rest/services/Common/SMC_GIS/FeatureServer/0). Available under [Public domain by judicial interpretation of the Public Records Act](https://casetext.com/case/county-of-santa-clara-v-superior-court-10). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- [Santa Barbara County](https://services8.arcgis.com/s7n9cRiugyMCsR0U/ArcGIS/rest/services/Parcel_layers_ArcGISonline_LUZO/FeatureServer/0). Available under [Public domain by judicial interpretation of the Public Records Act](https://casetext.com/case/county-of-santa-clara-v-superior-court-10). Distributed by OpenAddresses. Accessed: 2025-03-09.
+- [Santa Clara County](https://data.sccgov.org/Government/AddressPoint/k4vk-5ggi). Available under [Public domain by judicial interpretation of the Public Records Act](https://casetext.com/case/county-of-santa-clara-v-superior-court-10). Distributed by OpenAddresses. Accessed: 2025-03-09.
```

Comment:
Data url is no longer active and the rest endpoint is no longer available. https://web.archive.org/web/20230208203041/https://data.sccgov.org/Government/AddressPoint/k4vk-5ggi is still available though.

Reply:
This is one of those weird ones that is hiding its service directory but the data is still available through api calls.